plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_fr.properties

```diff
@@ -8,12 +8,12 @@ dataTransfer.processor.csv.description                          = Exporter en fi
 dataTransfer.processor.csv.name                                 = CSV
 dataTransfer.processor.csv.property.delimiter.description       = S\u00E9parateur de colonnes
 dataTransfer.processor.csv.property.delimiter.name              = S\u00E9parateur
-dataTransfer.processor.csv.property.escape.description          = Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '\' )
-dataTransfer.processor.csv.property.escape.name                 = Echappement de caract\u00E8res
+dataTransfer.processor.csv.property.escape.description=Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '' )
```

Comment:
```suggestion
dataTransfer.processor.csv.property.escape.description=Mod\u00E8le d'\u00E9chappement de caract\u00E8res sp\u00E9ciaux (entourr\u00E9s de guillemets ou \u00E9chapp\u00E9s par le caract\u00E8re '\\' )
```

Reply:
Return the correct value from the original English message.

Reply:
Reverted